### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesEdit_EditRecipe_ScreenReader: After changing the value for review, voice over is not announcing new value, it’s reading previous value.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15622]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_iOS_May23; #.NET Core; #WCAG1.3.1; #iOS; #iPhone; #Voiceover;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: iOS (iPhone 6s)
Screen Reader: Voiceover

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till any existing recipe and double tap.
4. "Recipe" Screen will open.
5. Swipe till "Edit" button and double tap.
6. Start Voice over.
7. Swipe till plus/minus button and change the value and observe after changing the value for review, voice over is not announcing new value, it’s reading previous value.

### Actual Results:
After changing the value for review, voice over is not announcing new value, it’s reading previous value.

**Similar issue is also observed in below scenario:
Issue 1: Recipes App My Recipes New_ New Recipe:**

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Start Voice over.
6. Swipe till plus/minus button and change the value and observe after changing the value for review, voice over is not announcing new value, it’s reading previous value.

### Expected Results:
After changing the value for review, voice over should announce new value which is been updated.

### User Impact:
Screen reader users will not get correct information about the updated value and will get confused as they have changed the value but it is announcing previous value.

### Attachment:

https://github.com/dotnet/maui/assets/90901519/06fd054c-0ff2-4a7e-a9d0-88523d987692

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS version 15.7.6

**Comment by rachelkang** [comment]
I'm not seeing this issue anymore, so marking as fixed. Will send updated app for re-testing when it's available.

**Comment by PoojaNamde** [comment]
@rachelkang 

Please provide the latest build of application once its available to verify this issue.

**Comment by PoojaNamde** [comment]
@rachelkang 
Please provide the latest build of application once its available to verify this issue.

**Comment by PoojaNamde** [comment]
#Regressed:08-17-23; #Closed;

Verified the bug in latest environment.
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
